CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Each release can have sections: "Added", "Changed", "Deprecated", "Removed", "Fi
 
 ### Added
 
+- GEDI L2B Satellite class. It is a seperate class and file to GEDI L2A data for now, they could be merged under a common GEDI parent class.
```

Comment:
Fix typos, add links to commit.
I prefer to be descriptive in the CHANGELOG : *add support for GEDI L2B product*, no more.